e2e/shortcut-action/shortcut-action.po.ts

```diff
@@ -49,4 +53,12 @@ export class ShortcutActionPage extends Ng2JsonEditorPage {
     let elem = element(by.id(id));
     return elem.all(by.css('input')).count();
   }
+
+  getElementArrayById(id: string) {
```

Comment:
AFAI understand from the code, it returns values of children inputs. So maybe a more descriptive name like `getValueOfChildrenById`.